File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/Related.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Related {

    @SerializedName("recommended")
    @Expose
    private Recommended recommended;
    @SerializedName("public-transport")
    @Expose
    private PublicTransport publicTransport;

    public Recommended getRecommended() {
        return recommended;
    }

    public void setRecommended(Recommended recommended) {
        this.recommended = recommended;
    }

    public PublicTransport getPublicTransport() {
        return publicTransport;
    }

    public void setPublicTransport(PublicTransport publicTransport) {
        this.publicTransport = publicTransport;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/Related.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class Related {
    @SerializedName("recommended")
    @Expose
    private var recommended: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Recommended? = null

    @SerializedName("public-transport")
    @Expose
    private var publicTransport: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.PublicTransport? = null
    fun getRecommended(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Recommended? {
        return recommended
    }

    fun setRecommended(recommended: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Recommended?) {
        this.recommended = recommended
    }

    fun getPublicTransport(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.PublicTransport? {
        return publicTransport
    }

    fun setPublicTransport(publicTransport: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.PublicTransport?) {
        this.publicTransport = publicTransport
    }
}